src/js/tracks/text-track-list-converter.js

```diff
@@ -28,8 +28,11 @@ let trackToJson_ = function(track) {
         id: cue.id
       };
     }),
-    src: track.src
   };
+  if (track.src) {
```

Comment:
Yeah, I was debating that. We do know exactly what keys we want, so, we could just loop through those keys and copy them over only if they have values. We might need to do a null check on the other side.